src/model/user.model.js

```diff
@@ -0,0 +1,63 @@
+import {v4 as uuidv4} from 'uuid'
+
+const users = [];
+
+module.exports = class User {
+    constructor(login, password, age) {
+        this.id = uuidv4();
+        this.login = login;
+        this.password = password;
+        this.age = age;
+        this.isDeleted = false;
+    }
+
+    save() {
+        users.push(this);
+    }
+
+    static getById(id) {
+        for (let user of users) {
+            if (user.id === id) {
+                return user;
+            }
+        }
+    }
+
+    static getAutoSuggestUsers(loginSubstring, limit) {
+        let count = 0;
+        let result = [];
+
+        for (let user of users) {
```

Comment:
Please check js array methods, they are much more clear and better for performance.
https://javascript.info/array-methods

Here you can use methods: filter and slice